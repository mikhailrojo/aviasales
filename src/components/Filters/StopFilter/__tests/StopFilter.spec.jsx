import * as React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import {StopFilter} from '../index';

describe('Snapshot', () => {
	it('render StopFilter with only last option selected', () => {
		const renderer = new ShallowRenderer();
		const component = renderer.render(
			<StopFilter
				filterByStops={jest.fn()}
				chosenStops={[false,false,false,true]}
			/>
		);
		expect(component).toMatchSnapshot();
	});
});
