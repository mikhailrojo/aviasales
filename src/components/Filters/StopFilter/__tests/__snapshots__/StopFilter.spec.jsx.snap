// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot render StopFilter with only last option selected 1`] = `
<styled.div>
  <styled.div>
    Количество пересадок
  </styled.div>
  <Checkbox
    checked={false}
    label="Все"
    onChange={[Function]}
    style={
      Object {
        "display": "flex",
      }
    }
    value="all"
  />
  <Checkbox
    checked={false}
    label="Без пересадок"
    onChange={[Function]}
    style={
      Object {
        "display": "flex",
      }
    }
    value="0"
  />
  <Checkbox
    checked={false}
    label="1 пересадка"
    onChange={[Function]}
    style={
      Object {
        "display": "flex",
      }
    }
    value="1"
  />
  <Checkbox
    checked={false}
    label="2 пересадки"
    onChange={[Function]}
    style={
      Object {
        "display": "flex",
      }
    }
    value="2"
  />
  <Checkbox
    checked={true}
    label="3 пересадки"
    onChange={[Function]}
    style={
      Object {
        "display": "none",
      }
    }
    value="3"
  />
</styled.div>
`;
